legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/legend-engine-xt-relationalStore-core-pure/src/main/resources/core_relational/relational/pureToSQLQuery/pureToSQLQuery.pure

```diff
@@ -6436,7 +6436,7 @@ function <<access.private>> meta::relational::functions::pureToSqlQuery::process
   let paths = $mainQuery.select->cast(@TdsSelectSqlQuery).paths;
   
   // todo this should be the behavior, but just scoping variant to reduce scope of change
-  let newElement = if($expressionToRoute->isVariantInput($vars, $state),
+  let newElement = if($expressionToRoute->isVariantInput($vars, $state) || ($expressionToRoute->instanceOf(SimpleFunctionExpression) && $expressionToRoute->cast(@SimpleFunctionExpression).func->toString() == 'if_Pair_MANY__Function_1__T_m_'),
```

Comment:
can cast to FunctionExpression and don't need the string conversion:

$expressionToRoute->instanceOf(FunctionExpression) && $expressionToRoute->cast(@FunctionExpression).func == if_Pair_MANY__Function_1__T_m_